packages/components/nodes/chatmodels/SonixAi/SonixAi.ts

```diff
@@ -0,0 +1,132 @@
+import { ICommonObject, INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses, getCredentialData, getCredentialParam } from '../../../src/utils'
+import { ChatOpenAI, OpenAIChatInput } from 'langchain/chat_models/openai'
+import { SonixAudioTranscriptionLoader } from "langchain/document_loaders/web/sonix_audio";
+import {SupportedLanguage}  from 'sonix-speech-recognition/lib/types'
+import { BaseCache } from 'langchain/schema'
+import { BaseLLMParams } from 'langchain/llms/base'
+
+class SonixAi implements INode {
+    label: string
+    name: string
+    version: number
+    type: string
+    icon: string
+    category: string
+    description: string
+    baseClasses: string[]
+    credential: INodeParams
+    inputs: INodeParams[]
+
+    
+    constructor() {
+        this.label = 'SonixAi'
+        this.name = 'SonixAi'
+        this.version = 1.0
+        this.type = 'SonixAi'
+        this.icon = 'sonixAi.svg'
+        this.category = 'Chat Models'
+        this.description = 'Converts audio to transcription'
+        this.baseClasses = [this.type, ...getBaseClasses(SonixAudioTranscriptionLoader)]
+        this.credential = {
+            label: 'Connect Credential',
+            name: 'credential',
+            type: 'credential',
+            credentialNames: ['sonixAIApi']
+        }
+        this.inputs = [
+            {
+                label: 'Cache',
+                name: 'cache',
+                type: 'BaseCache',
+                optional: true
+            },
+            {
+                label: 'Audio File',
+                name: 'audio_File',
+                type: 'file',
+                fileType: '.*'
+            },{
+                label: 'Audio File Name',
```

Comment:
why so we need to specify audo file name? maybe we can just get the file name from `audio_File`?

Reply:
I removed audio filename as input and took that from uploaded audio file in next commit